qemu/qemu.go

```diff
@@ -391,6 +412,9 @@ func (netdev NetDevice) QemuParams(config *Config) []string {
 	var qemuParams []string
 
 	deviceParams = append(deviceParams, fmt.Sprintf("%s", netdev.Driver))
+	if netdev.DisableModern {
```

Comment:
We need to change the way we do network devices.
We need to switch to the new format
``
"driver=virtio-net-pci,netdev=%s,mac=%s
```